### PR TITLE
Task module Functionality test cases and scenarios added

### PR DESCRIPTION
**Pull request by haliderdur**
This is done work for Task module functionality

### CODE REVIEW

**Comment by ismail-ozkan** [comment]
seems good to go

**Comment by gorkemagac** [comment]
I liked it so much. Brilliant job !!

File: src/test/java/com/ceallo/pages/TaskPage.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    @FindBy(xpath = "//div[@class='title']")
+    public WebElement newTaskTitle;
+
+    @FindBy(xpath = "//div[@class='title']")
+    public WebElement completedTaskTitle;
+
+    @FindBy(xpath = "//div[@class='title']")
+    public WebElement importantTaskTitle;
+
+
```
Comment (gorkemagac) [inline src/test/java/com/ceallo/pages/TaskPage.java:77]: Very organized. Good Job.

**Review by ismail-ozkan — APPROVED** [review]

**Review by gorkemagac — COMMENTED** [review]
Approved

**Pull request merged**